{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -115,6 +115,10 @@ matrix:
           env: ASTROPY_VERSION=development NUMPY_VERSION=development
                EVENT_TYPE='cron'
 
+        # Try on Windows.
```

Comment:
this may need to be templated with a "need windows testing?" or similar. But I don't have strong preferences, actually I'm not sure how many people are using, and then reusing cookiecutter config.

Reply:
So, should I add this option in or not, @bsipocz and @astrofrog ?

Reply:
I'd say no, I don't see the harm in just including it?